### PR TITLE
English translation

### PR DESCRIPTION
**Pull request by maximevaillancourt**
Feel free to review and propose changes.

### CODE REVIEW

File: index-en.html [hunk 1]
```diff
@@ -0,0 +1,849 @@
...
+                    <div class="row">
+                        <div class="col-md-6 col-sm-6">
+                            <div class="about-box wow animated fadeInDown" data-wow-duration="0.3s">
+                                <div class="about-box-icon">
+                                    <i class="fa fa-lightbulb-o"></i>
+                                </div>
+                                <div class="about-box-text">
+                                    <h4>JDIS Live! </h4>
+                                    <p align="justify">
+                                        JDIS (French for "Jeux et Défis Informatiques de Sherbrooke", literally "Sherbrooke Computer Science Games and Challenges") is a student group at Université de Sherbrooke which is strongly interested in computer science. It is also an AGEG technical group. Whether you are an engineering student or a science student of computer science, or simply a computer science geek, you are more than welcome among us.
```
Comment (DrPandemic) [inline index-en.html]: > a science student of computer science

a computer science student

File: index-en.html [hunk 2]
```diff
@@ -0,0 +1,849 @@
...
+                                    </p>
+                                    <p align="justify">
+                                        The group was created at the beginning of summer of 2015 and progressively takes shape. We are currently working on multiple projects, none of which you'll want to miss! Stay tuned to be notified of our progress and to learn more about our events, competitions, meetups and conferences related to computer science at Université de Sherbrooke.
```
Comment (DrPandemic) [inline index-en.html]: > the beginning of summer of 2015

sounds weird

File: index-en.html [hunk 3]
```diff
@@ -0,0 +1,849 @@
...
+                        </div>
+                        <div class="col-md-6 col-sm-6">
+                            <div class="about-box wow animated fadeInDown" data-wow-delay="0.5">
+                                <div class="about-box-icon">
+                                    <i class="fa fa-code"></i>
+                                </div>
+                                <div class="about-box-text">
+                                    <h4>Our mission</h4>
+                                    <p align="justify">
+                                        Our core mission is the create a strong community of students who share a passion for computer science. Such a community allows us to help our members reach their goals, whether they're about skillset development, professional development, involvement, personal projects, general discoveries, and more!
```
Comment (DrPandemic) [inline index-en.html]: > is the create

is to create

File: index-en.html [hunk 4]
```diff
@@ -0,0 +1,849 @@
...
+                    <div class="row">
+                        <div class="col-md-6 col-sm-6">
+                            <div class="about-box wow animated fadeInDown" data-wow-delay="0.5s">
+                                <div class="about-box-icon">
+                                    <i class="fa fa-image"></i>
+                                </div>
+                                <div class="about-box-text">
+                                    <h4>Our vision</h4>
+                                    <p align="justify">
+                                        We want to be at the center of everything related to computer science, here at the main campus of Université de Sherbrooke. We are a catalyst for events, participation and implication, as well as an accelerator to help our members reach the industry's standards of excellency.
```
Comment (DrPandemic) [inline index-en.html:124]: > campus of Université

sounds weird

File: index-en.html (hunk 1 above)
Comment (maximevaillancourt) [inline index-en.html]: I see where you're coming from, but keep this in mind: we want to include students from both the engineering and the science faculties. The original text makes it clear: "Que vous soyez étudiant en informatique _à la Faculté des sciences ou à la Faculté de génie_ ...".

File: index-en.html (hunk 2 above)
Comment (maximevaillancourt) [inline index-en.html]: How about "... at the beginning of summer 2015 ..."?

File: index-en.html (hunk 3 above)
Comment (maximevaillancourt) [inline index-en.html]: Good catch!

File: index-en.html (hunk 4 above)
Comment (maximevaillancourt) [inline index-en.html:124]: I was wondering whether we should keep the original name (Université de Sherbrooke) or translate it loosely (University of Sherbrooke / Sherbrooke University). That would help with the rest, such as "campus of [the] Sherbrooke University", for example.

File: index-en.html (hunk 1 above)
Comment (DrPandemic) [inline index-en.html]: Yes sure but the text is 

> Whether you are an engineering student or a science student of computer science

My proposition is

> Whether you are an engineering student or a computer science student

or could also be

> Whether you are an engineering student or a science student

I should have been clearer

File: index-en.html (hunk 2 above)
Comment (DrPandemic) [inline index-en.html]: :100: times better

File: index-en.html (hunk 4 above)
Comment (DrPandemic) [inline index-en.html:124]: Yesterday we were speaking about this, and we concluded that on the university web site they don't translate it. So we won't too.

**Comment by maximevaillancourt** [comment]
@DrPandemic: there, all fixed. :)

**Pull request merged**